### ISSUE TITLE
Conversations About Data - Offering a concept as feedback on the recently launched next.data.gov

### ISSUE THREAD

**Issue by edmullen** [GSA/datagov-wptheme#36]
**I spent a bit of time going through and thinking about the [Next.Data.gov](http://next.data.gov) launch and thought I'd add some ideas into the mix. It sort of coincides with something that I've felt has been an underutilized tactic in the evangelization of data. I hope this concept is helpful in some way and look forward to any discussion it may provoke.**

[**The concept can also be viewed here.**](http://www.edmullen.com/ideas/conversations-about-data)

[Download the Balsamiq wireframe](http://www.edmullen.com/files/conversation-page.bmml.zip).

_(Note: I'm rather new to Github. As a designer, I feel like a bit of an outsider, so please forgive any faux pas. I think there is a lot of potential for designers using Github, but I'm just getting started.)_ 

---

At times we may assume the value of specific data are self-evident, when they may not be. This misses an opportunity to expand the audience for the data beyond those already interested. People who are not deeply immersed in a subject matter community, but nonetheless could bring new ideas to the table, may benefit from some data literacy.

A very simplified life cycle for government data:
1. The data is collected
2. The data is released
3. The value of the data is understood
4. The data is used

We’ve been good at the first step for a long time.

In the last few years we’ve made great strides in the second step. [Data.gov](http://www.data.gov) and related data portals provide unprecedented access to data that can be reused in innumerable ways.

There has been lots effort put into the fourth step, advocating and incentivizing for the use of government data through the [Health Datapalooza](http://www.healthdatapalooza.org), challenges, and other efforts. [Next.data.gov](http://www.next.data.gov) (and [alpha.data.gov](http://alpha.data.gov) before it) does a great job of profiling successful use of data by entrepreneurs and folks in the private sector.

**The third step is one that I think could use some more love.** Helping people to see the value in a specific data set could help start the gears turning in the minds of creative people who may not immediately understand the contents of a new release.

I’ve put together a rough concept of a “Conversation” page for next.data.gov that would accompany prominent or undervalued data releases to provide context. These conversations are intended to help “unpack” the raw data from multiple perspectives; helping those who might use or share the data understand the potential value in it.

This example is based on the Inpatient Medicare Provider Charge Data. All the copy is for demonstration purposes only. It’s based on a rough understanding of the data and is likely to be inaccurate. But hopefully it helps articulate the concept.

![conversation-page](https://f.cloud.github.com/assets/2654503/957868/db12fc14-0459-11e3-9ff7-2971c8c30f9b.png)

**Comment by nickolasnikolic** [comment]
Quite honestly, I regret that I can't emphasize how much I agree with you on your thoughts beyond the suggestion that I am patting you on the back mentally and offering a digital hand to shake.

Good for you for voicing the thought here.

There are lots of strong feelings about what to believe, but very few factual traces of why to believe it. It is to help be involved with this very notion of spreading the vectors of the array of perspectives as to why and how data is interpreted that I am here. It is ultimately a question of proof, perspective, equal access, and ultimately truth.

Incidentally, there also is not even a remote doubt as to the importance usability plays in how successfully rich the audience is for this project.

I personally thank you for being here and if you find that you are getting even moderately picked on by any 'techie' user of this site, send them my way.., I'll take personal responsibility to defend your important perspective as your rhetorical shield :-)

Edit: Now that we have established that everyone is here because they care :-) It might be worth noting that there is an angle to this particular discussion from a more liberal arts perspective on Quora:

https://www.quora.com/Government/How-should-we-continue-to-improve-Data-gov

Where some of the consumers of the data speak on what they expect of the future of this project, including the topic of accessibility to the meaning of the data and how to improve it. (note of bias: I have answered, as well.)

**Comment by edmullen** [comment]
Thanks for the back-pat, @nickolasnikolic. I'm glad it struck a chord with you. My only real question was over what the best method for inserting a more conceptual bit of feedback within the existing culture of GitHub, sort of disconnected from the existing build. Submitting an issue felt closest. 

When projects like this are released and there are requests involvement from the community, it is such an opportunity. I hope to see more and more involvement from more and more people, where the challenge becomes managing too many good ideas.

**Comment by rypan** [comment]
This is a fantastic idea. I think this addresses a concern we've been having for a while. We release so many datasets (past and present) that are gems, but it takes a 5 minute conversation to show why. This idea scales our "conversations".

@edmullen Is it the job of the data steward to come up with the questions? Or is that a community driven piece. The community asks questions - the data stewards respond. Or do you imagine this being a highly editorial "post"?

**Comment by edmullen** [comment]
@rypan Good question. I think some exploration might be necessary. It would be good to define the audience and need a bit more. 

If we assume these are folks with a general interest but not specific expertise, at first, they may not have a perspective on the data from which to ask questions. Some general standard questions may catch their interest. Basic Who?, What?, When?, Where?, Why? -type questions could be a good start. The idea would be for data stewards or external folks who already understand the value could help explain what's so special about it. 

There may also be a need to respond to more pointed questions from people who already sort of get it, but still have some. 

By having a period of open-questions through the comment section or via some social media outreach, some follow-up questions could surface. They could either be answered in the comments if the question is of narrow value, or added into the general body of the page afterward. I don't think the Q&A needs to be a faithful record of an event. If there is a gap that is discovered afterward, I think it could be added later. So it may end up that over the coarse of a few weeks the page starts out with somewhat standard, pre-planned questions and grows a bit into a resource that includes public questions.

In the end, the hope would be that the "conversation" could aid in the initial launch but also serve as an ongoing resource whenever people stumble upon it.

In general, I think really general "Do you have any questions about Hospital Charge Rates?" prior to release wouldn't lead to really meaningful questions that would be generally helpful. I think the conversation should be started by people who can guide it a bit.

(Sidenote: I now think the page could benefit from a mechanism allowing people to jump down to questions.)

**Comment by JeanneHolm** [comment]
Really great idea and interested in thinking about the intersection of this type of dataset-specific conversation and Open Data Stack Exchange http://opendata.stackexchange.com/, which has a Q&A feel but could be on a wide variety of open data issues.  Some of the conversations there are about specific datasets (see http://opendata.stackexchange.com/questions/742/additional-detail-from-ipeds-data-source). I wonder if starting with some of our featured datasets on Next.Data.gov might be a way to go: http://highlights.data.gov/

**Comment by edmullen** [comment]
Yeah, starting with some of the existing data makes sense, or testing the idea with new high-profile data sets as part of a launch. The Open Data Stack Exchange is great, but it feels like it comes after this initial Q&A. The thinking is that the internal/industry experts really understand the value of the data in deep ways and the Q&A is sort of just a writing device to draw out the things they already know but that others wouldn't know to ask. 

In the sketch above, I included simple commenting at the bottom. But maybe instead you could start out with the expert dialogue, then at the end (and probably mention it at the top) shift saying "Ask your own questions on Open Data Stack Exchange" and then move folks over to that space. That may be more effective than commenting.
